README.md

```diff
@@ -16,16 +16,19 @@ for details.
 
 * Thrustmaster T500RS
 * Thrustmaster TX
-
+* Thrustmaster TMX
 See the status of additional devices on
 [GitHub Projects](https://github.com/her001/tmdrv/projects/1).
 
 ## Requirements
 
 * [Python 3](https://www.python.org)
 * [python-libusb1](https://pypi.python.org/pypi/libusb1)
-* jscal (from [Linux Console](https://sourceforge.net/projects/linuxconsole),
-for calibration)
+* jscal 
+ - Ubuntu: sudo apt install joystick
+ - Other Distro's: compile from source from [Linux Console](https://sourceforge.net/projects/linuxconsole),
+for calibration) 
+
```

Comment:
I agree that the readme could use better installation instructions, and your alternative is much more attractive, but I am not ready to accept such a change in this pull request. Please open a separate issue where we can discuss this.